### PR TITLE
add crystal2nix and colmena

### PR DESCRIPTION
**Pull request by peterhoeg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +93,10 @@
 
 ## Programming Languages
 
+### Crystal
+
+* [crystal2nix](https://github.com/nix-community/crystal2nix) - Convert shard.lock into Nix expressions.
```
Comment (houstdav000) [inline README.md:98]: Add backticks/grave symbols around "shard.lock" for formatting consistency with other filenames like `poetry.lock`.

File: README.md [hunk 2]
```diff
@@ -59,6 +60,7 @@
 
 ## Deployment Tools
 
+* [colmena](https://github.com/zhaofengli/colmena) - A simple, stateless NixOS deployment tool modeled after NixOps and morph.
```
Comment (houstdav000) [inline README.md:63]: Since [the project](https://github.com/zhaofengli/colmena) capitalizes "Colmena", we should too. Otherwise, looks good!

**Review by houstdav000 — CHANGES_REQUESTED** [review]
In the future, I'd recommend keeping entries separated mostly for ease of review (and ease of merging), but two is manageable.

I'd recommend slight changes to the entries for formatting consistency, but otherwise they're good.

As for the projects, Colmena looks pretty fleshed out, while I think crystal2nix might best add some to the `README.md`, particularly on the usage (what the command will do) and examples.

**Comment by peterhoeg** [comment]
> while I think crystal2nix might best add some to the README.md, particularly on the usage (what the command will do) and examples.

There's quite a few things that will be added to this over the next couple of weeks - documentation is one of them.

**Review by houstdav000 — APPROVED** [review]

**Pull request merged**